production/ksonnet/loki/config.libsonnet

```diff
@@ -40,6 +40,8 @@
     overrides_configmap_mount_name: 'overrides',
     overrides_configmap_mount_path: '/etc/loki/overrides',
 
+    jaeger_reporter_max_queue: 5000,
```

Comment:
Q: Curious what's the rationale behind this increase?

Reply:
Sorry this is from https://grafana.com/blog/2020/07/09/where-did-all-my-spans-go-a-guide-to-diagnosing-dropped-spans-in-jaeger-distributed-tracing/

Reply:
Sorry there's no much context here, but the idea is thatI want to make it easier to set the queue size for tracing as we can see that Loki generally creates a lot of span that can be dropped.

May be the default is not so great though, I could tune down to 1k and do 5k internally for us.

Reply:
@dannykopping wdyt ?

Reply:
Yea!. 5k may be too much for default!